posydon/tests/utils/test_configfile.py

```diff
@@ -57,7 +57,7 @@ def test_IO(self):
 
         # check `print` and `str` methods
         print(str(self))
-        print(len(str(self).split("\m")))
+        print(len(str(self).split("\m")))   # TODO: Should this be r"\m"?
```

Comment:
I'd say this depends on whether the `str()` call on the object reruns a raw string or not. For me it's not that obvious what it returns, but I'd guess the standard wouldn't be a raw string, hence `"\m"` should be correct.

In fact `"\m"` and `r"\m"` return the same object, hence it won't make a difference. But e.g. `r"\t"` is equivalent to `"\\t"` instead of `"\t"`, thus here it would be important to know, whether the string object is a raw string or not when using the split function on it.

Reply:
I got linter warning on that - so maybe there are cases where this is important. I suggest to change it to r"\m" if this is what we want to be the string exactly. I was just not sure about the intention in this part of the code.

Reply:
Let's resolve this with a separate PR. The TODO statement allows us to find it in the future.